lib/msprime.c

```diff
@@ -133,17 +133,16 @@ msp_strerror(int err)
         case MSP_ERR_BAD_RECOMBINATION_MAP:
             ret = "Bad recombination map provided.";
             break;
-        case MSP_ERR_BAD_COALESCENCE_RECORDS_SAMPLE_SIZE:
-            ret = "Bad coalescence records in file: Samples must be labeled 0...n-1 and not be internal nodes.";
+        case MSP_ERR_BAD_NODES_SAMPLE_SIZE:
+            ret = "Bad nodes in file: No sampled nodes.";
```

Comment:
Technically we need at least 2 samples I think, so maybe we should change this to ``MSP_ERR_INSUFFICIENT_SAMPLES`` and change the error message? Probably the "Bad nodes in file" bit doesn't add much and can be removed.